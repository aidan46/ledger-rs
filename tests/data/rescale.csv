type, client, tx, amount
deposit, 1, 1, 1.46
deposit, 1, 2, 2.12345
deposit, 1, 3, 2.2e2
