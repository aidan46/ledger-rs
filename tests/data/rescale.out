client,available,held,total,locked
1,223.5835,0,223.5835,false
